apps/src/templates/progress/progressStyles.js

```diff
@@ -65,28 +106,30 @@ const statusStyle = {
  * Given a level object, figure out styling related to its color, border color,
  * and background color
  */
-export const levelProgressStyle = (level, disabled) => {
+export const levelProgressStyle = (levelStatus, levelKind, disabled) => {
```

Comment:
For the new component, would it make sense to fold this logic into ProgressTableLevelBubble?  (This seems to be in the same vein as the logic in ProgressTableLevelBubble?)

Reply:
this actually _is_ the logic used by ProgressTableLevelBubble to determine border and background color. it's also used by some of the legacy components, but when we refactor those i agree that it would be better to move this into whatever component ends up being the common styling point between them.